examples/test/model/index.html

```diff
@@ -11,13 +11,16 @@
       <a-assets>
```

Comment:
Could also add the different format types in the page title & meta description `Models (glTF, COLLADA, OBJ)`